src/hotspot/share/opto/superword.hpp

```diff
@@ -506,8 +506,20 @@ class SuperWord : public ResourceObj {
   int adjacent_profit(Node* s1, Node* s2);
   int pack_cost(int ct);
   int unpack_cost(int ct);
+
   // Combine packs A and B with A.last == B.first into A.first..,A.last,B.second,..B.last
   void combine_packs();
+
+  // Split packs that are too long
+  void split_packs_for_max_vector_size();
```

Comment:
How about naming this `split_packs_longer_than_max_vector_size()`?

Reply:
done.